health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FileComponent.js

```
@@ -1,89 +1,130 @@
-                    {/* Right side: Edit, Delete, and Audit details */}
-                    <div className="dm-actions-container">
+    return (
+        <Card
+            style={{ background: "#FAFAFA", border: "1px solid #D6D5D4" }}
+        >
+            <div className="dm-parent-container" style={{ background: "#FAFAFA", margin: "0" }}>
+                <div
+                    className="dm-uploaded-file-container-sub"
+                    style={{ marginLeft: "-1rem" }}
+                >
+                    <div onClick={async () => await handleFilePreview(rowDetails?.fileStoreId)}>
```

Biome

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)